### DISCUSSION TITLE
How can we search multiple userId in the search filter of api/reg/search API in dockstaging?

### DISCUSSION THREAD

**Discussion by githubpiyush** [Sunbird-coKreat/Community#77] [Q&A] [answered]
@reshmi-nair
cc :- @aks30
API
`https://dockstaging.sunbirded.org/api/reg/search`

Body
`{ "id": "open-saber.registry.search", "ver": "1.0", "ets": "11234", "params": { "did": "", "key": "", "msgid": "" }, "request": { "entityType": [ "User_Org" ], "filters": { "userId": { "eq": "d59873a0-40ea-461b-9402-ab090932f92d" } } } }
`
In the above body we are searching for only one userId.
Is there any way to search for multiple users in one search filter?
If yes please share the filter.

**Comment by arun1205** [comment]
@githubpiyush looking into this

**Reply by githubpiyush** [reply]
@arun1205 Okay

**Reply by arun1205** [reply]
@githubpiyush

```
 "filters": {
            "userId": {
                "contains": ["4ff7d0f7-0d3f-4fa2-9756-c384b5c665c2"]
            }
        }
```

try this once

**Reply by githubpiyush** [reply]
> @githubpiyush
> 
> ```
>  "filters": {
>             "userId": {
>                 "contains": ["4ff7d0f7-0d3f-4fa2-9756-c384b5c665c2"]
>             }
>         }
> ```
> 
> try this once

It's not working @arun1205

**Reply by githubpiyush** [reply]
@arun1205 It's not working please recheck.

**Reply by githubpiyush** [reply, marked as answer]
`
  "filters": {
    "userId": {
      "or": [
        "fdfe6a85-14e0-498e-9663-07d9072aba91",
        "d59873a0-40ea-461b-9402-ab090932f92d"
      ]
    }
  }`
 
This filter is working for me @arun1205 thanks!
